firepit/sqlstorage.py

```diff
@@ -960,7 +960,63 @@ def timestamped(
         else:
             res = qry
         return res
+    def getAllNestedObjectsWithAnAttributeOfSCO(
```

Comment:
- Please leave 2 blank lines before each function definition.
- All other method names use snake case (https://en.wikipedia.org/wiki/Snake_case), not camel case.  Please follow the existing style.
- The method name is way too long, and I don't really understand what it means.